integration/fuse/src/test/java/alluxio/fuse/AlluxioFuseFileSystemTest.java

```diff
@@ -168,14 +169,17 @@ public void create() throws Exception {
     mFileInfo.flags.set(O_WRONLY.intValue());
     mFuseFs.create("/foo/bar", 0, mFileInfo);
     AlluxioURI expectedPath = BASE_EXPECTED_URI.join("/foo/bar");
-    verify(mFileSystem).createFile(expectedPath);
+    verify(mFileSystem).createFile(expectedPath, CreateFilePOptions.newBuilder()
+        .setMode(new alluxio.security.authorization.Mode((short) 0).toProto())
```

Comment:
@LuQQiu  will creating a file with "000" permission bits gets problem? Or, step back, is 0 the right permission level to test?

Reply:
In local map, I am able to use `unmask 777; touch 1.file` and get a `1.file` without permission.
In mac and Alluxio, 000 are all allowed permission.

Reply:
If the given mode is invalid, at least mac will directly reply error message. Invalid mode will not be passed to AlluxioFuse.